docs/source/install.rst

```diff
@@ -1,42 +1,145 @@
+Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords using an scripted installation or step by step commands.
+
+Scripted Installation
+----------------------
+The scripted installation utilizes Ansible to install the command line and container image and any associated dependencies.
+
+Installing the Ansible Prerequisite
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Follow the `Ansible installation documentation <http://docs.ansible.com/ansible/latest/intro_installation.html#installing-the-control-machine>`_ to prepare the your system to install the software.
+
+Obtaining the Installer
+^^^^^^^^^^^^^^^^^^^^^^^
+After Ansible is installed, you can obtain the installer that enables the use of the Quipucords command line and server.
+
+1. Download the installer by entering the following command::
+
+    # curl -k -O -sSL https://github.com/quipucords/quipucords/releases/download/1.0.0/quipucords.install.tar.gz
+
+2. Unzip the installer with the following command::
+
+    # tar -xvzf quipucords.install.tar.gz
+
+Executing the Installer
+^^^^^^^^^^^^^^^^^^^^^^^
+You can install in different ways, depending on your needs:
+
+- You can run with internet connectivity which will download necessary packages or setup and pull for needed repositories. This choice is the recommended approach because it simplifies the installation.
+
+- You can download the RPM packages and container image and place them in a specified directory. This choice is useful when installing on systems with limited or no access to the internet.
+
+Installing with Internet Connectivity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You are now prepared to run the installer.
+
+1. Change into the installer directory with the following command::
+
+    # cd install/
+
+2. To start the installation you can run the following command::
+
+    # ./install.sh
+
+The output appears similar to the following example::
+
+    Installation complete.
+
+Installing with Downloaded Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You must first gather the necessary packages.
+
+In order to install the server you will need the following items, see `Installing the Quipucords Server Container Image`_ for download locations for specific operating systems:
+
+- Docker package
+- Server container image
+
+In order to install the command line you will need the following items, see `Installing the Quipucords Command Line Tool`_ for download locations for specific operating systems:
+
+- EPEL package
+- QPC package
+
+1. Change into the installer directory with the following command::
+
+    # cd install/
+
+2. Create a directory, default is ``packages``, to install downloaded packages from with the following command::
```

Comment:
default is ``packages`` --> the recommended default is ``packages`` --> This line isn't incorrect, it just reads weird to me so feel free to leave it as it is.

Reply:
I took this out since the `pkg_install_dir` option tries to explain how you could use a different directory with the installer.